include/deal.II/base/iterator_range.h

```diff
@@ -220,13 +244,26 @@ class IteratorRange
   IteratorOverIterators
   begin();
 
+//  /**
+//   * Return the iterator pointing to the first element of this range.
+//   */
+//  const IteratorOverIterators &
```

Comment:
?

Reply:
This is still a draft PR, and is labelled WIP. I don't know how else to  say that I'm still fiddling with this...

Reply:
You didn't put `WIP` in the caption. :wink: